ghdocs/Tutorial.md

```diff
@@ -0,0 +1,153 @@
+## Overview
+In this tutorial, let's build a simple react app with a DocumentCard fabric component.
+
+## Prerequisites
+### Node.js and npm
+Install [Node.js and npm](https://nodejs.org/en/) if you haven't already installed in your machine. It is recommended you at least install node `version 4.x.x` and npm `version 3.x.x`.
+
+>You can check your node and npm version by running `node -v` and `npm -v` respectively.
+
+### create-react-app package
+We will use [Create React App](https://facebook.github.io/react/blog/2016/07/22/create-apps-with-no-configuration.html) to quickly create a react app for the tutorial.
+
+Once node and npm is installed, type the following command in a console to install the `create-react-app` npm package.
+
+```
+npm install -g create-react-app
+```
+
+### Visual Studio Code
+Download and install [Visual Studio Code](https://code.visualstudio.com/).
+
+>We will use Visual Studio Code as our code editor for this tutorial. Feel free to use your favorite code editor of choice if you do not want to use Visual Studio Code.
+
+## Step 1: Create react app
+In a console, type the following command to create a simple react app.
+
+```
+create-react-app documentcardbasic-demo
+```
+
+This will install the required depedencies and scaffold a simple react app.
+
+![create-react-app command completed](./images/create-react-app-created.png)
+
+Once its complete, navigate to the `documentcardbasic-demo` folder.
+
+```
+cd documentcardbasic-demo
+```
+
+Type the following command to launch the react app in your browser.
+
+```
+npm start
+```
+
+![create-react-app preview in browser](./images/create-react-app-preview.png)
+
+## Step 2: Install office-ui-fabric-react package
+To use Office UI Fabric React components in your project, you will need to first install the ofifce-ui-fabric-react npm package. Type the following in the console to install the package:
```

Comment:
No need for the "Office UI" since we've already introduced the name - Fabric React is OK.